packages/components/psammead-play-button/CHANGELOG.md

```diff
@@ -0,0 +1,6 @@
+# Psammead Play Button Changelog
+
+<!-- prettier-ignore -->
+| Version | Description |
+| ------------- | ----------- |
+| 1.0.0-alpha.0 | [PR#2248](https://github.com/bbc/psammead/issues/2248) Create initial package. |
```

Comment:
I think this needs to link to the PR not the issue
```suggestion
| 1.0.0-alpha.0 | [PR#2253](https://github.com/bbc/psammead/pull/2253) Create initial package. |
```

Reply:
Another good spot. Thanks, Harvey.